tests/integration/views/test_session.py

```diff
@@ -9,3 +10,7 @@ def test_session_expired(self):
     def test_session_signed_out(self):
         self.get('/signed-out')
         self.assertInPage('Your survey answers have been saved')
+
+    def test_rrm_page(self):
```

Comment:
This names doesn't make sense. the assert below should just be moved into `test_session_signed_out`